include/dxc/DxilContainer/RDAT_LibraryTypes.inl

```diff
@@ -9,48 +9,19 @@
 //                                                                           //
 ///////////////////////////////////////////////////////////////////////////////
 
+// clang-format off
+// Macro indentation makes this file easier to read, but clang-format flattens
+// everything.  Turn off clang-format for this file.
+
 #ifdef DEF_RDAT_ENUMS
 
 RDAT_ENUM_START(DxilResourceFlag, uint32_t)
-RDAT_ENUM_VALUE(None, 0)
-RDAT_ENUM_VALUE(UAVGloballyCoherent, 1 << 0)
-RDAT_ENUM_VALUE(UAVCounter, 1 << 1)
-RDAT_ENUM_VALUE(UAVRasterizerOrderedView, 1 << 2)
-RDAT_ENUM_VALUE(DynamicIndexing, 1 << 3)
-RDAT_ENUM_VALUE(Atomics64Use, 1 << 4)
-RDAT_ENUM_END()
-
-RDAT_ENUM_START(DxilShaderFlags, uint32_t)
-RDAT_ENUM_VALUE(None, 0)
-RDAT_ENUM_VALUE(OutputPositionPresent, 1 << 0)
-RDAT_ENUM_VALUE(DepthOutput, 1 << 1)
-RDAT_ENUM_VALUE(SampleFrequency, 1 << 2)
-RDAT_ENUM_VALUE(UsesViewID, 1 << 3)
-RDAT_ENUM_VALUE(NodeProgramEntry, 1 << 4)
-RDAT_ENUM_END()
-
-RDAT_ENUM_START(NodeFuncAttribKind, uint32_t)
-RDAT_ENUM_VALUE(None, 0)
-RDAT_ENUM_VALUE(ID, 1)
-RDAT_ENUM_VALUE(NumThreads, 2)
-RDAT_ENUM_VALUE(ShareInputOf, 3)
-RDAT_ENUM_VALUE(DispatchGrid, 4)
-RDAT_ENUM_VALUE(MaxRecursionDepth, 5)
-RDAT_ENUM_VALUE(LocalRootArgumentsTableIndex, 6)
-RDAT_ENUM_VALUE(MaxDispatchGrid, 7)
-RDAT_ENUM_VALUE_NODEF(LastValue)
-RDAT_ENUM_END()
-
-RDAT_ENUM_START(NodeAttribKind, uint32_t)
-RDAT_ENUM_VALUE(None, 0)
-RDAT_ENUM_VALUE(OutputID, 1)
-RDAT_ENUM_VALUE(MaxRecords, 2)
-RDAT_ENUM_VALUE(MaxRecordsSharedWith, 3)
-RDAT_ENUM_VALUE(RecordSizeInBytes, 4)
-RDAT_ENUM_VALUE(RecordDispatchGrid, 5)
-RDAT_ENUM_VALUE(OutputArraySize, 6)
-RDAT_ENUM_VALUE(AllowSparseNodes, 7)
-RDAT_ENUM_VALUE_NODEF(LastValue)
```

Comment:
These three enums were moved to the section requiring 1.8 or above around line 174

Reply:
Just a comment to try to help reviewers. Can be resolved by anyone.